scripts/vm-guest-tools.ps1

```diff
@@ -78,4 +78,17 @@ if ("$env:PACKER_BUILDER_TYPE" -eq "virtualbox-iso") {
   cmd /c rd /S /Q "C:\Windows\Temp\virtualbox"
 }
 
+if ("$env:PACKER_BUILDER_TYPE" -eq "parallels-iso") {
+  Write-Output "Using Parallels Desktop"
+  # This is required when using Parallels Desktop
+  # Currently the installation of Parallels Tools is broken if you're installing in Windows Server Core
+  # You will encounter an error in printui.dll module not found message.
+  # This feature will add the required libraries for printing services.
+
+  # Parallels Tools is required for sync_folder to work.
+  # Vagrantfile config: v.update_guest_tools = true will be installed with no issues
+  Write-Output "Installing Print Server service"
+  Install-WindowsFeature -Name Print-Server
```

Comment:
Sorry, this command won't work on Windows 10, and this provision script is used for all Windows templates.

```
PS C:\Windows\system32> Install-WindowsFeature -Name Print-Server
Install-WindowsFeature : The term 'Install-WindowsFeature' is not recognized as the name of a cmdlet, function, script
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.
At line:1 char:1
+ Install-WindowsFeature -Name Print-Server
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Install-WindowsFeature:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Maybe something similar to https://github.com/StefanScherer/packer-windows/blob/main/scripts/debloat-windows.ps1#L18 could help to run this only on Windows Server.

Reply:
Ah, right, Im concentrated to work this on my workflow (WIndows Server) without checking out other versions of Windows.

Let me fix and test this @StefanScherer. Thanks!